ajax/libs/react-mdl/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "react-mdl",
+  "filename": "material.min.js",
+  "version": "1.10.3",
+  "description": "React Components for Material Design Lite",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/tleunen/react-mdl"
+  },
+  "homepage": "https://react-mdl.github.io/react-mdl/",
+  "keywords": [
+    "react",
+    "react-component",
+    "mdl",
+    "material-design",
+    "badge",
+    "button",
+    "card",
+    "checkbox",
+    "data-table",
+    "fab-button",
+    "icon",
+    "icon-button",
+    "icon-toggle",
+    "radio",
+    "radio-group",
+    "switch",
+    "textfield",
+    "progress",
+    "progress-bar",
+    "spinner",
+    "slider",
+    "tooltip",
+    "menu",
+    "dropdown",
+    "Layout",
+    "Drawer",
+    "Grid",
+    "Tabs"
+  ],
+  "author": {
+    "name": "Tommy Leunen",
+    "email": "tommy.leunen@gmail.com",
+    "url": "https://tommyleunen.com/"
+  },
+  "license": "MIT",
+  "npmName": "react-mdl",
+  "npmFileMap": [
+    {
+      "basePath": "extra",
```

Comment:
I think we need files in `out` folder rather than `extra`.
`material` in `extra` belong another lib.
https://github.com/react-mdl/react-mdl/blob/v1.10.3/extra/material.css#L6